src/helpers/shuffle.js

```diff
@@ -0,0 +1,7 @@
+export const shuffle = arr => {
```

Comment:
Normally in JS we try to write in full words (writing in shortened words is something that was done with older languages like C), so we should prefer to write `array` instead of `arr`.